extensions/wikia/Email/EmailController.class.php

```diff
@@ -644,4 +644,13 @@ private static function getLegendName() {
 
 		return $legendName;
 	}
+
+	/**
+	 * A wrapper for wfMessage() which removes the possibility of messages being overridden in the MediaWiki namespace
+	 *
+	 * @return Message
+	 */
+	protected function getMessage() {
+		return wfMessage( func_get_args() )->useDatabase( false );
```

Comment:
I tested it with the Forgot Password email and it worked. You might be right though. I also may have misread how func_get_args() could be used in the PHP manual. I'll investigate and fix if necessary.